pypeit/core/flexure.py

```diff
@@ -668,23 +667,23 @@ def spec_flexure_slit(slits, slitord, slit_bpm, sky_file, method="boxcar", speco
             continue
 
         # get spectral FWHM (in Angstrom) if available
```

Comment:
please change this comment: `(in Angstrom)`--> `(in pixels)`, just to avoid confusion

Reply:
Good catch - updated!